server/index.js

```diff
@@ -22,6 +23,20 @@ app.use('/test', (req, res) => {
   res.send('3-2-1 testing! Server is serving!');
 });
 
+app.get('/:id', (req, res) => {
+  if (req.params.id > 100 || req.params.id < 1) {
```

Comment:
While this will help with out of range numbers, it may make sense to add something like `parseInt(req.params.id) !== parseInt(req.params.id) || ` to the conditions here to cover ids that aren't numbers (forcing them to NaN).

Reply:
Assuming you mean `req.params.id !== parseInt(req.params.id)`. Added for future PR.

Reply:
No, I meant the above, to take advantage of the fact that `(NaN !== NaN) === true`.